2.9.0/done/result_xpu.md

```diff
@@ -27,20 +27,29 @@ The categories below are as follows:
 ### bc breaking
 ### deprecation
 ### new features
+- Enable `FlexAttention` on Intel GPU ([#143553](https://github.com/pytorch/pytorch/pull/143553))
+- Enable `_int_mm` on Intel GPU ([#157769](https://github.com/pytorch/pytorch/pull/157769))
+
 ### improvements
+- Add UUID to Intel GPU device properties ([#161392](https://github.com/pytorch/pytorch/pull/161392))
+- Support Intel GPU quantization ops in AOTInductor ([#156572](https://github.com/pytorch/pytorch/pull/156572))
+- Enable tensor memory descriptor Triton template for Intel GPU ([#161600](https://github.com/pytorch/pytorch/pull/161600))
+- Support `FlightRecorder` for XCCL to allows debug trace dumps for distributed jobs on Intel GPU ([#158568](https://github.com/pytorch/pytorch/pull/158568))
+
 ### bug fixes
+- Fix cpp_extension compatibility with oneAPI DPC++ 2025.2 compiler ([#161012](https://github.com/pytorch/pytorch/pull/161012))
+
 ### performance
+- Integrate OneDNN SDPA training forward and backward on Intel GPU ([#161058](https://github.com/pytorch/pytorch/pull/161058))
+
 ### docs
+- Update supported OS to Windows 11 & Ubuntu 24.04/25.04 for Intel client GPU ([#161699](https://github.com/pytorch/pytorch/pull/161699))
+
 ### devs
+- Upgrade Intel OneDNN commit pin to v3.9.1 ([#161932](https://github.com/pytorch/pytorch/pull/161932))
+- Upgrade Intel Triton commit pin to 3.5.x ([#161777](https://github.com/pytorch/pytorch/pull/161777))
+- Upgrade Intel GPU support package to OneAPI 2025.2 ([#158733](https://github.com/pytorch/pytorch/pull/158733))
```

Comment:
Upgrade Intel GPU software stack package to intel-deep-learning-essentials-2025.2